cookbooks/lib/features/basic_spec.rb

```diff
@@ -221,8 +223,14 @@ def empty_dir
     its(:exit_status) { should eq 0 }
   end
 
-  describe command(%(eval "$(HOME=#{Support.tmpdir} gimme 1.6.3)" 2>&1)) do
-    its(:stdout) { should match 'go version go1.6.3' }
+  if os[:arch] !~ /ppc64/
+    describe command(%(eval "$(HOME=#{Support.tmpdir} gimme 1.6.3)" 2>&1)) do
+      its(:stdout) { should match 'go version go1.6.3' }
+    end
+  elsif os[:arch] =~ /ppc64/
+    describe command(%(eval "$(HOME=#{Support.tmpdir} gimme 1.6.4)" 2>&1)) do
+      its(:stdout) { should match 'go version go1.6.4 linux/ppc64le' }
+    end
```

Comment:
I'm happy to collapse these two and _only_ check the installation of `1.6.4` (or something much newer!) although this change can definitely come later...